integration-tests/src/runTest.ts

```diff
@@ -3,12 +3,31 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 
+import * as cp from 'child_process'
 import * as path from 'path'
-
-import { runTests } from '@vscode/test-electron'
+import {
+  downloadAndUnzipVSCode,
+  resolveCliArgsFromVSCodeExecutablePath,
+  runTests
+} from '@vscode/test-electron'
 
 async function main (): Promise<void> {
   try {
+    const vscodeExecutablePath = await downloadAndUnzipVSCode('1.84.2')
+    const [cliPath, ...args] = resolveCliArgsFromVSCodeExecutablePath(vscodeExecutablePath)
+
+    cp.spawnSync(
+      cliPath,
+      [
+        ...args,
+        '--install-extension', 'mads-hartmann.bash-ide-vscode@1.39.0',
```

Comment:
It looks like you didn't update the debug launch configuration in consequence. Have a look at .vscode/launch.json name "Integration Tests"

Reply:
Seems it won't work.

The one in launch.json will launch the integration tests into the user's environment, which we don't control. In my case, my other extensions will add diagnostics and make bitbake-parse.test.ts fail on lines [74](https://github.com/savoirfairelinux/vscode-bitbake/blob/a3615e6559149b1ef88c85a7449abad358abbe1e/integration-tests/src/tests/bitbake-parse.test.ts#L74) and [103](https://github.com/savoirfairelinux/vscode-bitbake/blob/a3615e6559149b1ef88c85a7449abad358abbe1e/integration-tests/src/tests/bitbake-parse.test.ts#L103). It seems not possible to deactivate all the extensions except the two we want. 

Since it is not reliable, I'd suggest we delete it.

Reply:
This configuration is immensely useful for debugging integration tests. I suggest to remove the "--disable-extension" from it and document installing the dependencies on the host vscode with "code --install-extension" commands in integration-tests/README.md.